tests/contrib/litellm/conftest.py

```diff
@@ -2,33 +2,88 @@
 
 from ddtrace.contrib.internal.litellm.patch import patch
 from ddtrace.contrib.internal.litellm.patch import unpatch
+from ddtrace.llmobs import LLMObs as llmobs_service
+from ddtrace.llmobs._constants import AGENTLESS_BASE_URL
+from ddtrace.llmobs._writer import LLMObsSpanWriter
 from ddtrace.trace import Pin
 from tests.contrib.litellm.utils import get_request_vcr
 from tests.utils import DummyTracer
-from tests.utils import DummyWriter
+from tests.utils import override_global_config
+
+
+class TestLLMObsSpanWriter(LLMObsSpanWriter):
```

Comment:
Just an FYI: this is probably going to be moved as a central test util class after https://github.com/DataDog/dd-trace-py/pull/12966/ gets merged, so we can just reuse that once you're back from PTO